tests/test_webeditorposts.py

```diff
@@ -256,16 +233,15 @@ def test06_test_publish_button(self):
             btn_publish = soup.findAll("button", {"id": "btn_publish"})
             assert btn_publish and btn_publish[0].text.strip() == "Publish"
 
+    @unittest.skip("post deletion not implemented")
     def test07_test_delete_button(self):
-        """Test webpost deletion."""
-        rv = self.client.get('/delete_post?post_id={}'.format(self.post_id))
+        """Test post deletion."""
+        rv = self.client.get('/delete_post?path={}'.format(self.post_id))
         assert rv.status == "200 OK"
 
         with self.app.app_context():
-            post = (db_session.query(Post)
-                              .filter(Post.id == self.post_id)
-                              .first())
-            assert not post
+            kp = self.repo.post(self.post_path)
+            assert kp.is_deleted is True
```

Comment:
question - would `assert kp.is_deleted` be equivalent to `assert kp.is_deleted is True`